src/integrations/devicecontext.ts

```diff
@@ -1,47 +1,35 @@
-import { addEventProcessor, getCurrentHub } from '@sentry/core';
 import type { Contexts, Event, Integration } from '@sentry/types';
 import { logger } from '@sentry/utils';
 
 import { NATIVE } from '../wrapper';
 
-/** Load device context from native. */
-export class DeviceContext implements Integration {
-  /**
-   * @inheritDoc
-   */
-  public static id: string = 'DeviceContext';
+const INTEGRATION_NAME = 'DeviceContext';
 
-  /**
-   * @inheritDoc
-   */
-  public name: string = DeviceContext.id;
+export const deviceContextIntegration = (): Integration => {
+  return {
+    name: INTEGRATION_NAME,
+    setupOnce: () => {
+      // noop
+    },
```

Comment:
`setupOnce` in v8 is optional, we can drop it, in RN it was need as we migrated to the new integration style before v8.